User Guide/Projects.md

```diff
@@ -718,6 +718,12 @@ To set up **Web socket connection**:
 
 ![Image not Available](/assets/Fig200.png)
 
+## Websocket code sample ## 
+
+Here is the demo code samples related to Paho websocket related, custom business logic is not included. Download the files and use for reference, can use anyone of the platform e.g. java, python, .net fw, core, nodejs, or go.
```

Comment:
Here is the demo code sample with Paho library, custom business logic is not included. Download the file and use for reference.

Reply:
ok